javalib/src/main/scala/java/net/Socket.scala

```diff
@@ -248,7 +248,7 @@ class Socket protected (private[net] val impl: SocketImpl,
     val value = if (on) {
       if (linger > 65535) 65535 else linger
```

Comment:
This PR does not touch this line, but it would be a mercy if a one line comment
were to be placed just above it explaining Java 8 specifies the maximum
value as "The maximum timeout value is platform specific" but actual
practice is to limit to 65545.   It took me a while to figure out this magic
number (slow child) and others tracing the code may have the same 
difficulty.  A breadcrumb would save others effort.

Reply:
If you look at the linked FreeBSD source code above there is a check that the value is not > USHORT_MAX, so 65535 seems a safer max value.

Reply:
Probably it was a typo, I've added a comment.